styles/designs/carnival/parts/lists/_lists-components.scss

```diff
@@ -10,6 +10,14 @@ $UnorderedList--FirstLevel: (
     )
 );
 
+$UnorderedList--FirstLevel_FromStimulus: (
+    _name: "FirstLevelUnorderedListFromStimulus",
+    _subselector: ' > ul > li',
+    _properties:(
+        margin-left: h-spacing(3),
+    )
+);
+
```

Comment:
We don't need new component. It has the same properties as regular `$UnorderedList--FirstLevel`. We just have to put it under a proper wrapper in a shape.

Reply:
Thanks, I've removed the redundant component :-)